### ISSUE TITLE
Can Field be extended to return multiple densities instead of just one?

### ISSUE THREAD

**Issue by Serenitysmk** [nerfstudio-project/nerfstudio#1496]
Hi!

many thanks for such great work!

I am not reporting a bug issue here, I am just wondering whether I am able to extend `Field` to return multiple densities in `get_density()` function. For instance 3 sigmas for each RGB channel TensorType[..., 3] instead of just one sigma to represent the density?

thanks in advance for answering

Best regards,
mshe

**Comment by tancik** [comment]
I don't think that would be an issue. You would need to updated the code that uses the densities.
